components/brave_shields/browser/base_brave_shields_service.h

```diff
@@ -12,47 +12,37 @@
 #include <vector>
 #include <mutex>
 
-#include "base/sequenced_task_runner.h"
 #include "content/public/common/resource_type.h"
 #include "url/gurl.h"
 
 namespace brave_shields {
 
-class DATFileWebRequest;
-
 // The brave shields service in charge of checking brave shields like ad-block,
 // tracking protection, etc.
 class BaseBraveShieldsService {
  public:
-  BaseBraveShieldsService(const std::string& file_name, const GURL& url);
+  BaseBraveShieldsService();
   virtual ~BaseBraveShieldsService();
   bool Start();
   void Stop();
   bool IsInitialized() const;
   virtual bool ShouldStartRequest(const GURL& url,
       content::ResourceType resource_type,
       const std::string& tab_host);
-  scoped_refptr<base::SequencedTaskRunner> GetTaskRunner() {
-    return task_runner_;
-  }
+//  scoped_refptr<base::SequencedTaskRunner> GetTaskRunner() {
+//    return task_runner_;
+//  }
```

Comment:
This is used by the browser tests but those will need to be updated anyway. You can just remove the code for it.